src/pat/contentbrowser/src/ContentBrowser.svelte

```diff
@@ -426,11 +434,15 @@
                                 {#if level.selectable}
                                     <button
                                         class="btn btn-primary btn-xs"
+                                        data-bs-toggle="tooltip"
+                                        data-bs-title="{level.displayPath}"
+                                        data-bs-placement="top"
+                                        data-bs-custom-class="contentbrowser-tooltip"
```

Comment:
```suggestion
                                        title="{level.displayPath}"
```

Reply:
I like the idea of simplifying it. I will change it.